source/nodejs/README.md

```diff
@@ -13,13 +13,16 @@ This directory contains all the JS packages for adaptive cards:
 Removed JS packages:
 * `adaptivecards-fabric` - A set of Fabric UI extensions for the Adaptive Cards JS renderer. This package has been deprecated. Please migrate to [`adaptivecards-fluentui`](https://www.npmjs.com/package/adaptivecards-fluentui). The source code is available in [21.09](https://github.com/microsoft/AdaptiveCards/releases/tag/21.09) or before releases.
 
+## Authenticate with npm registry
+
+1. `vsts-npm-auth -config .npmrc` in `source/nodejs`
```

Comment:
Are the new auth requirements part of the switch to npm?

Reply:
We're required to use our internal artifact feed with public `npm` as an upstream (hit me up for a link if you want more info). This work should have been done some time ago, but it wasn't until I started messing with things that CI tooling started complaining. :)

Reply:
This is for ci pipelines, for example, if I want to work on JS, I don't need to do this step?